neuralmonkey/logging.py

```diff
@@ -14,8 +16,10 @@ class Logging(object):
 
     # 'all' and 'none' are special symbols,
     # others are filtered according the labels
-    debug_enabled = ['all']
-    debug_disabled = []  # type: List[str]
+    debug_enabled = [
+        os.environ.get("NEURALMONKEY_DEBUG_ENABLE", "none")]  # type: List[str]
+    debug_disabled = [
+        os.environ.get("NEURALMONKEY_DEBUG_DISABLE", "")]  # type: List[str]
```

Comment:
nevim, tak to bylo, tak jsem to jen předělal. Přijde mi to lepší tam mít, když chceš všechno kromě jednoho, tak se to bude psát líp. ale nebudu proti, až to někdo v budoucnu bude chtít vyhodit